opencti-platform/opencti-front/lang/fr.json

```
@@ -2519,8 +2516,35 @@
   "Lifecycle": "Cycle de vie",
   "Lifecycle details": "Détails du cycle de vie",
   "Subscription to specific object(s)": "Abonnement à un (des) objet(s) spécifique(s)",
+  "Max Confidence Level": "Niveau de confiance maximal",
+  "This value is overridden for the following entity types": "Cette valeur est remplacée pour les types d'entités suivants",
```

```suggestion
  "This value is overridden for the following entity types": "Cette valeur est remplacée pour les types d'entités suivantes",
```
I think that "entité" is feminine.

---

indeed ! FYI it's Deepl auto translate

---

hm actually it's "types suivants"